src/write_resume_data.cpp

```diff
@@ -141,24 +141,16 @@ namespace {
 				{
 					auto const& verified = atp.verified_leaf_hashes[f];
 					if (!verified.empty())
-					{
-						auto& ret_verified = ret_dict["verified"].string();
-						ret_verified.reserve(verified.size());
-						for (auto const bit : verified)
-							ret_verified.push_back(bit ? '1' : '0');
-					}
+						ret_dict["verified2"] = std::string(verified.data()
+							, std::size_t(verified.size() + 7) / 8);
```

Comment:
Wouldn't it be better to provide data size via `bitfield` interface?

Reply:
yeah, that would probably be better. there's already a member for the number of words